### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD004.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#8] [General]
# https://blog.3place.cn/3place/fsd/FSD004.html

https://blog.3place.cn/3place/fsd/FSD004.html

**Comment by picklecai** [comment]
这是你的世界！也算个世界！

是啊！这怎么能算个世界？这怎么能不是梅菲斯特说的蚱蜢？可是被圈在里面，无奈呀。

**Comment by picklecai** [comment]
天主给自然创造人类，
你却避开生动的自然。

原来缘木求鱼还能这么用。我们要“自然教育”，要直接体验，要感官感受……然后才能转动思维的马达。

**Comment by picklecai** [comment]
我觉得年轻神圣的生的幸福
重新热烈地流遍神经和脉管。

这句话写的是心流。只是不知道此时它来自哪里。

**Reply by picklecai** [reply]
从这些纯洁的笔划之间，
我看到创造的自然在我心灵前呈现。

**Comment by picklecai** [comment]
我是浮士德，我跟你平辈！

这话比孙悟空说得谦虚多了。悟空说：雷公是我的孙子，夜叉是我重孙，马面是我玄孙。

**Reply by picklecai** [reply]
浮士德对大地，没有敬仰。

**Comment by picklecai** [comment]
我就在轰轰的时间织机之旁
织造神的有生命的衣裳

有意思。交给时间，自有道理。

**Reply by picklecai** [reply]
生命来自于大地。

**Comment by picklecai** [comment]
像我们这样关在研究室里，
偶尔在节日才去观光世界，
只用望远镜远远观瞧，
怎能进行说服将世人指导？

瓦格纳说得很有道理。从浮士德的回答看，他俩一个经验主义，一个理性主义。

**Comment by picklecai** [comment]
要获得一种探本穷源的方法，
这是多么困难的事情！

浮士德一个人自言自语的苦恼，不也是这个吗？

**Comment by picklecai** [comment]
所谓认识，是什么情形！谁能直截了当地说出真理？

遇到梅菲斯特之前的浮士德，是不可知论。因为理性太奥妙，渺小的人类没有这种能力。

**Reply by picklecai** [reply]
因此我就向魔术献身，
想通过精灵的有力的口舌，
使我了解到许多秘密；
使我不需要再流酸汗，
把不懂的事信口乱谈；
使我认识是什么将万物
囊括于它的最深的内部，
看清一切动力和种子，
不再需要咬文嚼字。

浮士德的不可知论，是说人类不可知。他觉得非人类是可以知道这个世界的根本规律的。所以他希望把自己献给他们，借以换取对世界的洞若观火。这里有没有掌控感？不好说。《超体》的女主，《驴皮记》的男主……都是浮士德。

**Comment by picklecai** [comment]
满以为凭着创造，可以享受
神的生活，却偏偏自取其咎！
雷鸣般的语言夺去我的一切。

谁雷鸣般的语言？

**Comment by picklecai** [comment]
我们精神上最崇高的感受，
也常有各种杂质混入其中；
我们一达到世间的善境之后，
更善者就被称为空想和幻梦。

这段，非常柏拉图。这是金刚经为什么要将法也视为障碍之一的原因吧。

**Comment by picklecai** [comment]
你将为一切虚惊发抖，
还要为不会遇到的损失经常悲叹。

成仙成佛就要经受住这些考验。佛一语破之：一切都是幻相。

**Comment by picklecai** [comment]
复活节前夜，这个时间的设定也有意思。

**Comment by picklecai** [comment]
这最后一盅，让我全心全意，
作为酒宴的崇高的祝贺，献给晨光！

没想到这是毒酒。已经厌弃到要自杀的地步了，难怪愿意和魔鬼订立契约。
